instagram/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import SearchBarContainer from './components/Search/SearchBarContainer';
+import PostContainer from './components/Post/PostContainer';
+import CommentsContainer from './components/Comments/CommentsContainer';
+import './App.css';
+import dummyData from './dummy-data.js';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ dummyData, searchInput: '' });
+    }, 1000);
+    // this.setState({ dummyData, searchInput: '' });
+  }
+
+  handleSearchInput = event => {
+    this.setState({ searchInput: event.target.value });
+  };
+
+  searchPosts = event => {
+    event.preventDefault();
+    if (this.state.searchInput) {
+      this.setState({
+        dummyData: this.state.dummyData.filter(post => {
+          if (post.username.includes(this.state.searchInput)) {
+            return post;
+          }
+        })
+      });
+    } else {
+      this.componentDidMount();
+    }
+  };
+
+  render() {
+    // console.log(this.state);
+    if (!this.state.dummyData) {
```

Comment:
Nice job using a loading message while the user waits for the app to load. You'll use this concept a bunch especially when we get into working with more 3rd party APIs or when we have a backend that we're retrieving data from 👍